intercom_flutter/README.md

```diff
@@ -66,61 +66,20 @@ This plugin works in combination with the [`firebase_messaging`](https://pub.dev
 
 * First, implement [`firebase_messaging`](https://pub.dev/packages/firebase_messaging) and check if it works: https://pub.dev/packages/firebase_messaging#android-integration
 * Then, add the Firebase server key to Intercom, as described here: https://developers.intercom.com/installing-intercom/docs/android-fcm-push-notifications#section-step-3-add-your-server-key-to-intercom-for-android-settings (you can skip 1 and 2)
-* Add the following to your  `AndroidManifest.xml` file, so incoming messages are handled by Intercom:
-
-```
-    <service
-        android:name="io.maido.intercom.PushInterceptService"
-        android:enabled="true"
-        android:exported="true">
-        <intent-filter>
-          <action android:name="com.google.firebase.MESSAGING_EVENT" />
-        </intent-filter>
-    </service>
-```
-_just above the closing `</application>` tag._
-
-* Ask FireBaseMessaging for the FCM token that we need to send to Intercom, and give it to Intercom (so Intercam can send push messages to the correct device):
+* Ask FirebaseMessaging for the FCM token that we need to send to Intercom, and give it to Intercom (so Intercom can send push messages to the correct device), please note that in order to receive push notifications in your iOS app, you have to send the APNS token to Intercom. Below example uses [`firebase_messaging`](https://pub.dev/packages/firebase_messaging) to get either the FCM or APNS token based on the platform:
 
 ```dart
-String token = await FirebaseMessaging.instance.getToken();
-Intercom.sendTokenToIntercom(token);
+final firebaseMessaging = FirebaseMessaging.instance;
+final intercomToken = Platform.isIOS ? await firebaseMessaging.getToken() : await firebaseMessaging.getAPNSToken();
```

Comment:
let's keep this example only for Android as it is under the Android section.

Reply:
We can add this in the example app by including a button "send token to intercom".